### ISSUE TITLE
Checkboxes on Expired Active Leases page missing labels

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6785]
### Descriptive summary
On dev.nurax.samvera.org (Hyrax 5.0.1), the checkboxes on the `Manage Leases` page do not have labels and therefore do not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/embargoes?locale=en#expired and run the Site Improve browser extension

![expired_lease_checkbox](https://github.com/samvera/hyrax/assets/32644254/ef196871-76a7-4e87-8304-60ee8b8a80e2)

Related work
#6778 
#6783

**Comment by christhepianist** [comment]
Tested and confirmed.

**Comment by christhepianist** [comment]
No filesets in pg.nurax, so updated the lease for a fileset to complete tomorrow and will check again.

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
